### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by Yevhenii-Bezeha**
Add new validation, authorAvarar, improve services

### CODE REVIEW

File: src/app/components/form/form.component.scss [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
-  border-radius: 4px;
-}
+  border-radius: 6px;
+  padding-top: 45px;
+  padding-bottom: 25px;
+  padding-left: 15px;
+  padding-right: 15px;
+  position: relative;
+  margin-left: 20px;
+  margin-right: 20px;
```
Comment (MMaksymivUa) [inline src/app/components/form/form.component.scss:13]: use just
``` 
padding: ...;
margin: ...;
```

File: src/app/components/modal/modal.component.ts [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
-export class ModalComponent implements OnInit {
+export class ModalComponent implements OnInit, OnDestroy {
+  constructor(private modalService: ModalService) {}
+
+  ngOnInit(): void {
+    window.addEventListener('keydown', this.onEsc);
+  }
+
+  ngOnDestroy() {
+    window.removeEventListener('keydown', this.onEsc);
```
Comment (MMaksymivUa) [inline src/app/components/modal/modal.component.ts:17]: good job, but please use window from angular
```
@inject(WINDOW) private _window
```

File: src/app/components/form/form.component.scss (hunk 1 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/form/form.component.scss:13]: Changed

File: src/app/components/modal/modal.component.ts (hunk 2 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/modal/modal.component.ts:17]: Good. Used window from angular. Thanks )

**Pull request merged**